tests/kaia_test_account_map_test.go

```diff
@@ -112,17 +112,10 @@ func (a *AccountMap) Initialize(bcdata *BCData) error {
 func (a *AccountMap) Update(txs types.Transactions, txHashesExpectedFail []common.Hash, txBundlingModules []builder.TxBundlingModule, signer types.Signer, picker types.AccountKeyPicker, currentBlockNumber uint64) error {
 	incorporatedTxs, _ := builder_impl.ExtractBundlesAndIncorporate(txs, txBundlingModules)
 	for _, txOrGen := range incorporatedTxs {
-		if gen, ok := txOrGen.(builder.TxGenerator); ok {
-			// To simulate tx, the nonce given to generate is set to zero.
-			// This does not affect subsequent operations on the AccountMap state.
-			tx, err := gen.Generate(0)
-			if err != nil {
-				continue
-			}
-			txs = append(txs, tx)
+		tx, err := txOrGen.GetTx(0)
+		if err != nil {
+			return err
```

Comment:
This seems like a good enough continue, but what do you think?
```suggestion
			continue
```

Reply:
Wouldn't it be better to explicitly return an error so caller can fix on error, rather than implicitly skipping? Also, this function already propagates subroutine errors.

Reply:
I agree with the opinion. Thank you.